### PR TITLE
[13.0][MIG] stock

### PR DESCRIPTION
**Pull request by Yasaie**
Migration of `stock`
--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/stock/migrations/13.0.1.1/post-migration.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    fill_stock_putaway_rule_company_id(env.cr)
+    fill_stock_move_line_company_id(env.cr)
+    fill_stock_package_level_company_id(env.cr)
+    fill_stock_picking_type_company_id(env.cr)
+    fill_stock_production_lot_company_id(env.cr)
+    fill_stock_scrap_company_id(env.cr)
```
Comment (kos94ok-3D) [inline addons/stock/migrations/13.0.1.1/post-migration.py]: Why did you create six functions with the same code?
```
def fill_company_id(cr):
    table_list = [
        'stock_scrap',
        'stock_picking_type',
        'stock_production_lot',
        'stock_move_line',
        'stock_package_level',
        'stock_putaway_rule',
    ]
    for table in table_list:
        openupgrade.logged_query(cr, """
            UPDATE {table}
            SET company_id = ru.company_id
            FROM res_users ru
            WHERE ru.id = {table}.create_uid AND {table}.company_id IS NULL
        """.format(table=table))
```

**Comment by MiquelRForgeFlow** [comment]
@Yasaie  I also have worked a bit on it and found many changes that should be considered. If you don't mind, I will do a supersede of this PR later on to add them. Some of them are difficult to express via comment.

**Comment by Yasaie** [comment]
@MiquelRForgeFlow Sure add them, thanks

**Comment by MiquelRForgeFlow** [comment]
Superseded by https://github.com/OCA/OpenUpgrade/pull/2306.

**Pull request closed without merging**